Source/WebCore/editing/Editor.cpp

```diff
@@ -3648,7 +3648,8 @@ static inline void collapseCaretWidth(IntRect& rect)
 
 IntRect Editor::firstRectForRange(const SimpleRange& range) const
 {
-    range.start.document().updateLayout();
+    Ref document = range.start.document();
+    document->updateLayout();
```

Comment:
Why not just range.start.protectedDocument()->updateLayout()?

Reply:
Fixed.